docs/deploy.sh

```diff
@@ -45,9 +45,10 @@ git config user.name "Travis CI"
 git config user.email "$COMMIT_AUTHOR_EMAIL"
 
 # If there are no changes to the compiled out (e.g. this is a README update) then just bail.
-if git diff --quiet; then
-    echo "No changes to the output on this push; exiting."
-    exit 0
+# NOTE: Disabled since it also suppresses re-compiled tutorial links, etc.
+#if git diff --quiet; then
+#    echo "No changes to the output on this push; exiting."
+#    exit 0
```

Comment:
Yes, this suppressed an update due to the last PR merge, which means that our website stays out-of-date until someone actually changes docstrings.